package.json

```diff
@@ -12,15 +12,24 @@
   "bugs": {
     "url": "https://github.com/RecordReplay/protocol/issues"
   },
+  "files": [
+    "js",
```

Comment:
wouldn't we want to export the `ts` files?

Reply:
I don't think so, the package contains `.d.ts` files, which is all that's needed to use the package in a typescript project.